### ISSUE TITLE
how to change the basebone from resnet18 to mobilenet_v2

### ISSUE THREAD

**Issue by zhangyunming** [WXinlong/SOLO#87]
hi, how to change the basebone from resnet to mobilenet_v2 or shufflenet included in torchvision by  modify the config file,thanks !

**Comment by Adyoshkin** [comment]
@zhangyunming I have same problem. Do you have any ideas of solution?

**Comment by cjfcsjt** [comment]
@zhangyunming @Adyoshkin Have you found a solution? I had the same problem.

**Comment by Adyoshkin** [comment]
@cjfcsjt  I've just added [here](https://github.com/WXinlong/SOLO/blob/master/mmdet/models/anchor_heads/solov2_head.py#L336): 
`            if len(ins_label) == 0:
                ins_label.append(torch.zeros([mask_feat_size[0], mask_feat_size[1]],
                                             dtype=torch.uint8, device=device))
                label = 0
                ins_ind_label[label] = True
                grid_order.append(label)

            ins_label = torch.stack(ins_label, 0)`

**Comment by WXinlong** [comment]
@zhangyunming @adeshkin @cjfcsjt  I have just fixed this issue. Thanks.
